sleepy/sleepy_python/README.md

```diff
@@ -1,3 +1,5 @@
 # sleepy_python
 
-Let's talk about this
+Background : Python's interpreter for sys.stdout is default to disabled
```

Comment:
we should mention that Python is buffering stdout, not that it's disabled.  Since it's buffered, we have to flush the buffer to get the writes to be visible in Nomad.